### ISSUE TITLE
Improved Podium icon to look more like a podium and not a bar graph

### ISSUE THREAD

**Issue by Jarvey** [driftyco/ionicons#402]
Can you improve the podium icon to make it more clear that it's a podium and not a bar graph icon.
For example you can add  a trophy on top of the bars or add podium numbers 1,2,3. See attached example. Thanks!
![improved-ion-podium](https://cloud.githubusercontent.com/assets/5406216/16262985/57d79a58-3872-11e6-9592-3783423b92a6.png)

**Comment by dharamveer-camplus** [comment]
> Can you improve the podium icon to make it more clear that it's a podium and not a bar graph icon.
> For example you can add a trophy on top of the bars or add podium numbers 1,2,3. See attached examples (I like the numbers on top the most). Thanks!
> 
> ![podium 2](https://cloud.githubusercontent.com/assets/5406216/16419141/e571474a-3d4c-11e6-8bcb-37660640ef2c.png)
> ![improved-ion-podium](https://cloud.githubusercontent.com/assets/5406216/16262985/57d79a58-3872-11e6-9592-3783423b92a6.png)

Yes, Ionic need to work lot with respect icons they are not that appealing anymore, ionic icon just ruin the app ui feel
